custom_components/circadian_lighting/sensor.py

```diff
@@ -6,27 +6,19 @@
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity import Entity
 
-from . import CIRCADIAN_LIGHTING_UPDATE_TOPIC, DOMAIN
+from . import CIRCADIAN_LIGHTING_UPDATE_TOPIC, DEFAULT_PROFILE, DOMAIN
 
 ICON = "mdi:theme-light-dark"
 
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Circadian Lighting sensor."""
-    circadian_lighting = hass.data.get(DOMAIN)
-    if circadian_lighting is not None:
-        sensor = CircadianSensor(hass, circadian_lighting)
-        add_devices([sensor], True)
-
-        def update(call=None):
```

Comment:
I removed the service_call because I am not sure how to best do it once there are multiple sensors. Also I think it's not really needed. If you want frequent updates, just decease the interval?

Reply:
I'm not sure how much the service call is used, TBH. Off the top of my head I could imagine it would be useful with the `only_once` option where a user may want to control exactly when a light adjusts (e.g. any time there's motion).